src/remediations/controller.write.js

```diff
@@ -55,6 +55,27 @@ async function validateNewActions(add) {
     });
 }
 
+async function validateRemediationName(name, tenant_org_id, username, transaction) {
```

Comment:
I did consider the UNIQUE constraint but figured this was more consistent with the rest of the code. 

@vkrizan That's a good point actually.. Would the db constraint cause issues with existing data? And if we keep this current implementation, I don't think it would catch any existing duplicate names unless the remediation is updated at some point. Not sure if this is a good thing or not?